scripts/gtk-bundle-from-msys2.sh

```diff
@@ -148,9 +148,24 @@ _getpkg() {
 		# use @(gz|xz|zst) to filter out signature files (e.g. mingw-w64-x86_64-...-any.pkg.tar.zst.sig)
 		ls $cachedir/mingw-w64-${ABI}-${1}-${package_version}-*.tar.@(gz|xz|zst) | sort -V | tail -n 1
 	else
-		# -dd to ignore dependencies as we listed them already above in $packages and
-		# make pacman ignore its possibly existing cache (otherwise we would get an URL to the cache)
-		pacman -Sddp --cachedir /nonexistent mingw-w64-${ABI}-${1}
+		case "$1" in
+		"cairo")
+			# stick with cairo-1.18.4-1 until lagging issue resolved
+			# https://github.com/geany/geany-plugins/issues/1466
+			# https://gitlab.freedesktop.org/cairo/cairo/-/issues/905
+			echo "https://mirror.msys2.org/mingw/mingw64/mingw-w64-x86_64-cairo-1.18.4-1-any.pkg.tar.zst"
+			;;
+		"pango")
+			# stick with pango-1.56.3-2 until lagging issue resolved
+			# https://github.com/geany/geany/pull/4360
+			echo "https://repo.msys2.org/mingw/mingw64/mingw-w64-x86_64-pango-1.56.3-2-any.pkg.tar.zst"
```

Comment:
```suggestion
			echo "https://repo.msys2.org/mingw/mingw64/mingw-w64-${ABI}-pango-1.56.3-2-any.pkg.tar.zst"
```
Ditto

Reply:
why does this use ***repo**.msys2.org* but Cairo uses ***mirror**.msys2.org*?

Reply:
No reason just a copy and paste from another script, let's stick with mirror, I will commit a change